internal/webui/webui.go

```diff
@@ -0,0 +1,51 @@
+/*******************************************************************************
+ * Copyright 2021 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ *******************************************************************************/
+
+package webui
+
+import (
+	"net/http"
+	"strconv"
+	"time"
+
+	"github.com/lf-edge/edge-home-orchestration-go/internal/common/logmgr"
+)
+
+var (
+	assetFolder = "web"
+	uiPort      = 49153
+	log         = logmgr.GetInstance()
+)
+
+// Start starts the server for web UI
+func Start() {
+	s := &http.Server{
+		Handler:      uiHandler(),
+		Addr:         ":" + strconv.Itoa(uiPort),
+		WriteTimeout: 15 * time.Second,
```

Comment:
Instead of having read/write timeout constant defined multiple places within code, defining it as constant variable at top or in environment variables can help in future maintainability

Reply:
Thanks for the attentive review! Done.